inlanecruising_plugin/src/inlanecruising_plugin.cpp

```diff
@@ -60,10 +60,53 @@ bool InLaneCruisingPlugin::onSpin()
   return true;
 }
 
+void InLaneCruisingPlugin::set_yield_client(ros::ServiceClient& client)
+{
+  yield_client_ = client;
+}
+
+bool InLaneCruisingPlugin::plan_trajectory_cb2(cav_srvs::PlanTrajectoryRequest& req,
+                                              cav_srvs::PlanTrajectoryResponse& resp)
+{
+
+  ROS_ERROR("in cb2");
```

Comment:
Is this actually an error?

Reply:
If this method is just for the unit/integration testing, I suppose this doesn't matter too much